packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { assert, details as X } from '@agoric/assert';
+import { sameStructure } from '@agoric/same-structure';
+
+import { buildParamManager, ParamType } from '../../../src/paramManager';
+
+const MALLEABLE_NUMBER = 'MalleableNumber';
+
+export const governedParameterTerms = {
```

Comment:
In #3310 the treasury declares two paramManagers for the parameters that relate to the AMM and to the Pools. Each Pool has a separate set of parameters, so the number of parameters being managed grows with the number of Pools.

The description of the parameters is exported so that the contract can refer to it.

Reply:
I think this example still has an unnecessary level to it. I'm not understanding why two sets of parameters requires the additional level.